docs/getting_started.rst

```diff
@@ -31,20 +31,19 @@ by ds9. Ruler, Compass and Projection types are ignored.
 Read Region Files
 =================
 
-*pyregion.open* takes the region name as an argument and returns a
-ShapeList object, which is basically a list of Shape objects. ::
+`pyregion.open` takes the region name as an argument and returns a
+`~pyregion.ShapeList` object, which is basically a list of Shape objects. ::
```

Comment:
If we do make it part of the API docs, should it be exposed as `pyregion.parser_helper.Shape` like it is now, or re-exposed from `pyregion.Shape`?